src/UI/TxDetails/index.tsx

```diff
@@ -58,31 +41,36 @@ export const TxDetails = ({
   };
   return (
     <>
-      {title && <div className={generatedClasses.title}>{title}</div>}
-      <div className={generatedClasses.statusTransactions}>
+      {title && <div className={styles.title}>{title}</div>}
+
+      <div className={styles.status}>
         {
           transactions.filter((tx) => !isServerTransactionPending(tx.status))
```

Comment:
I would wrap this whole part inside a memo because it looks like it performs a loop during each rerender

```
const foo = useMemo(() => {
const processedTransactions = transactions.filter((tx) => !isServerTransactionPending(tx.status)).length;
const totalTransactions = transactions.length;

return `${processedTransactions} / ${totalTransactions} transactions processed`;
}, [transactions.length])
```